index.js

```diff
@@ -2341,25 +2341,24 @@
   //. ```
   S.lastIndexOf = sanctifyIndexOf('lastIndexOf');
 
-  //# pluck :: Accessible a => TypeRep b -> String -> [a] -> [Maybe b]
+  //# pluck :: Accessible a => (b -> Boolean) -> String -> [a] -> [Maybe c]
```

Comment:
The idea would be that `Predicate a` somehow constrains `a` to be the set of values for which the predicate function of type `Predicate a` returned true.. It doesn't quite work like that I think, but I wish it could.